in_command.go

```diff
@@ -182,3 +190,19 @@ func (c *InCommand) downloadFile(url, destPath string) error {
 
 	return nil
 }
+
+func (c *InCommand) resolveTagToCommitSHA(tag string) string {
+	reference, err := c.github.GetRef(tag)
+
+	if err != nil {
+		fmt.Fprintln(c.writer, "could not resolve tag '%s' to commit: %v", tag, err)
+		return ""
+	}
+
+	if *reference.Object.Type != "commit" {
+		fmt.Fprintln(c.writer, "could not resolve tag '%s' to commit: returned type is not 'commit' - only lightweight tags are supported", tag)
```

Comment:
Can we have this case just not result in the file being created at all, rather than being blank? Better to have the file always be valid if it exists. People using annotated tags will probably understand why it's not there.